README.md

```diff
@@ -60,7 +50,7 @@ The startup script will do this for you so just run this command:
 
 When starting for the first time, or after a clean, the frontend will take a few minutes to start as node needs to install all the dependencies.
 
-To find out if the application has started, open a console window on the frontend docker container and view the logs. If the server has started successfully you will see this message `Server listening on port 3000`.  If this does not appear try forcing node to restart by updating one of the `.njk` files.
+To find out if the application has started, open a console window on the frontend docker container and view the logs. If the server has started successfully you will see this message `Server listening on port 3000`. If this does not appear try forcing node to restart by updating one of the `.njk` files.
```

Comment:
Is it worth explaining here why a change will cause Node to restart. Something along the lines of 

> ...because Node.js is watching files, any changes to files will cause Node.js to restart. If you don't see the `Server listening on port 3000` message, you can force a restart by running `touch src/server.ts` as a Terminal command or manually editing a file in the `src` directory

Reply:
100% agree, but that should be a separate PR